libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -130,6 +137,20 @@ bool AP_ESC_Telem::get_temperature(uint8_t esc_index, int16_t& temp) const
     return true;
 }
 
+// get the highest ESC temperature in centi-degrees if available, returns true on success
+bool AP_ESC_Telem::get_highest_temperature(int16_t& temp) const
+{
+    uint8_t valid_escs = 0;
+
+    for (uint8_t i = 0; i < ESC_TELEM_MAX_ESCS; i++) {
+        if (get_temperature(i, temp)) {
+            valid_escs++;
+        }
+    }
+
+    return !!valid_escs;
```

Comment:
I'm assuming this is a typo. Better to compare to 0.

Reply:
Not a typo. `!!x` is commonly used to convert an integer to boolean, it is equivalent to `x != 0`

Reply:
Ok I stand corrected!